### DISCUSSION TITLE
Is the VAD model locale agnostic?

### DISCUSSION THREAD

**Discussion by NitishG20** [snakers4/silero-vad#383] [Q&A] [answered]
I see the example shows an en language audio. Just wanted to confirm if the VAD model is locale agnostic or not? Or is it just expected to work well on a particular set of languages?

**Comment by snakers4** [comment, marked as answer]
Hi,

The VAD was trained on 100 languages, 10 languages being best represented.
The new version will be trained on many more languages.

**Reply by NitishG20** [reply]
Thanks for the quick response. 2 things: 1. Where can I find the 100 and 10 language list? 2. What is the date planned for the new version that you mentioned?
